terraform/api_gateway.tf.json.template.py

```diff
@@ -258,6 +285,17 @@ def for_domain(cls, domain):
                                     'action': {
                                         action: {}
                                     },
+                                    **(
+                                        {
+                                            # Add a label to these logs so they
+                                            # can be filtered from the log group
```

Comment:
```suggestion
                                            # Label these logs for filtering
```

I'm unsure that 'these' are logs, aren't they a set of IPs (allowed, or disallowed)?

Reply:
Either way, I don't think the comments should be verbose.